src/Features/LanguageServer/Protocol/ExternalAccess/Razor/ProjectContextsParams.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.Serialization;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.CodeAnalysis.LanguageServer.ExternalAccess.Razor;
+
+internal record ProjectContextsParams : ITextDocumentParams
```

Comment:
This actually simplifies from the VS version, where it uses `TextDocumentItem`. I don't forsee any fallout from that change though, and prevents us from passing the full text of the document back and forth